style.css/seventhsectionStyle.css

```diff
@@ -0,0 +1,23 @@
+.seventh-section {
+  text-align: center;
+}
+
+.seventh-section-header {
+  text-align: center;
+  background-color: #fa382f;
+  color: #ffffff;
+  display: inline-block;
+  font-weight: bold;
+  padding: 8px;
+  margin-top: 40px;
+}
+
+.for-more {
+  display: flex;
+  justify-content: space-around;
+}
+
+.for-more-item {
+  color: #fa382f;
+  margin: 40px 40px;
```

Comment:
This margin cause vertical scroll in mobile view. Please remove the left and the right. And for space add gap: 16px; to the ul.

Reply:
Done.

Reply:
Done.